client/lib/sites-list/list.js

```diff
@@ -167,31 +166,8 @@ SitesList.prototype.createSiteObject = function( site ) {
 		return Site( site );
 	}
 };
-/**
- * Marks collisions between .com sites and Jetpack sites that have the same URL
- * Add the hasConflict attribute to .com sites that collide with Jetpack sites.
- *
- * @api private
- *
- */
-SitesList.prototype.markCollisions = function( sites ) {
-	sites.forEach( function( site, index, collisions ) {
-		var hasCollision;
-
-		if ( ! site.jetpack ) {
-			hasCollision = some( collisions, function( someSite ) {
-				return (
-					someSite.jetpack &&
-					site.ID !== someSite.ID &&
```

Comment:
The only thing I'm thinking about is whether we need this or not? I don't think there's a chance for 2 sites to be jetpack and not-jetpack but share the same ID.

Reply:
the selector version which @enejb [linked](https://github.com/Automattic/wp-calypso/blob/5f15f3365213253724ca0d8798c6cb8160f00bc9/client/state/sites/selectors.js#L124-L148) has done away with the IDs altogether, and after seeing his description it seemed like they were irrelevant.